package-lock.json

```diff
@@ -34354,6 +34354,21 @@
     "tooling/typescript": {
       "name": "@isomer/tsconfig",
       "version": "0.0.0"
+    },
+    "tooling/template/node_modules/@next/swc-win32-ia32-msvc": {
```

Comment:
what's this change btw?

Reply:
idk, this seems to be an autogenerated change every time...